README.md

```diff
@@ -52,33 +52,33 @@ https://examples.contributte.org/planette/forest-project/
 
 1) At first, use composer to install this project.
 
-    ```
-    composer create-project -s dev planette/forest-project
-    ```
+	```bash
+	composer create-project -s dev planette/forest-project
+	```
 
 2) After that, you have to setup Postgres >= 10 database. You can start it manually or use docker image `postgres:10`.
 
-    ```
-    docker run -it -p 5432:5432 -e POSTGRES_PASSWORD=forest -e POSTGRES_USER=forest postgres:10
-    ```
+	```bash
+	docker run -it -p 5432:5432 -e POSTGRES_PASSWORD=forest -e POSTGRES_USER=forest postgres:10
+	```
 
     Or use make task, `make loc-postgres`.
 
 3) Custom configuration file is located at `app/config/config.local.neon`. Edit it if you want.
 
     Default configuration should look like:
 
-    ```yaml
-    # Host Config
-    parameters:
+```neon
```

Comment:
Tady jsme prisli o odsazeni, prosim vratit. Zaroven projdi cely readme, je to na vice mistech.